### DISCUSSION TITLE
Fehlende Werte im HSPU Dashboard

### DISCUSSION THREAD

**Discussion by Steinmetz1957** [Trunks1982/Daikin-Rotex-HPSU-CAN#60] [Q&A]
Hallo, danke für das tolle Projekt.
Kann mir jemand erklären warum in meiner Anzeige vom HSPU Dashboard so viele Angaben fehlen? Ich denke ich habe alles auf dem neuesten Stand. Aber wie kann ich das kontrollieren? In ESPHome und auch im Webserver sehe ich keine Versionsangabe.
![Screenshot 2024-11-08 114858](https://github.com/user-attachments/assets/e5749258-51a0-4349-8542-06bbedb7c583)

**Comment by Trunks1982** [comment]
Hi, die fehlenden Werte werden nur sichtbar wenn man noch die serielle Schnittstelle abfragt.Die anderen Werte sind alles can.

**Reply by Steinmetz1957** [reply]
OK alles klar. Danke für die schnelle Antwort.

**Comment by wrfz** [comment]
Das liegt daran, dass du bisher nur die CAN-Komponente verwendest. 
Wenn du auch Werte aus dem Kältekreis einbinden möchtest,
benötigst du zusätzlich die [Serielle Komponente](https://github.com/wrfz/esphome-components). 

Die serielle Komponente wird ähnlich wie die CAN-Komponente in deiner YAML-Datei referenziert.
Du kannst sie entweder auf demselben ESP oder auf einen separaten flashen.
In beiden Fällen musst du den ESP mit der seriellen Schnittstelle der HPSU verbinden (H1P Platine).
Leider haben wir noch keine detaillierte Anleitung dazu.

**Reply by aDabbelju** [reply]
Gibt es denn stichpunktartige Anleitung? Die serielle Anbindung hatte ich bereits schonmal mit einem separaten ESP angebunden über iobroker, aber der ist irgendwie abgestorben :(  Wenn ich es wieder zum Laufen bekomme, würde ich es auch dokumentieren.

**Reply by wrfz** [reply]
Bisher gibt es leider noch keine Dokumentation für die Seriell Komponente.
Der Ablauf ist allerdings derselbe wie bei der CAN-Komponente:
1) Im ESPhome ein yaml erstellen
2) Den Inhalt aus dem [examples/full.yaml](https://github.com/wrfz/esphome-components/blob/main/examples/full.yaml) in dein yaml Datei kopieren
3) GPIOs im yaml entsprechend deiner ESP-Verdrahtung konfigurieren
4) ESP über ESPhome flashen

Auf der [Startseite](https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN-Seriell) in diesem Projekt findest du ganz unten einen Link zu einer Telegram-Gruppe. Dort werden dir sämtliche deine Fragen beantwortet.

**Comment by wrfz** [comment]
@Steinmetz1957 Besteht das Problem noch? Oder kann das Thema geschlossen werden?

**Reply by Steinmetz1957** [reply]
Problem besteht weiter. Werde mich aber erst im neuern Jahr weiter damit beschäftigen.
Ich hab die Diskussion erst einmal geschlossen. Danke für die bisherigen Antworten.
